src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -153,19 +158,41 @@ const VideoPlayer: FC<VideoPlayerProps> = (props) => {
     }
   };
   const onError = (evt: Event) => {
+    const networkError =
+      (evt as CustomEvent).detail?.data?.type === "networkError";
+    // Don't report network errors.
+    if (networkError) {
+      return;
+    }
+
+    const newReloadOnErrors = [...reloadOnErrors];
+    const prevReloadOnErrorsLength = reloadOnErrors.length;
+    if (newReloadOnErrors.length < RELOAD_ATTEMPTS) {
+      const timeElapsed = getTimeElapsed(mediaElRef) || 0;
+      newReloadOnErrors.push(timeElapsed || 0);
+      setReloadOnErrors(newReloadOnErrors);
+    }
+
+    const hasMaxAttempts = newReloadOnErrors.length === RELOAD_ATTEMPTS;
+    const isFirstAttempt = newReloadOnErrors.length === 1;
+    const willReload = prevReloadOnErrorsLength !== newReloadOnErrors.length;
+    // Only report the first and last error
+    const shouldReport =
+      hasMaxAttempts || (willReload && (isFirstAttempt || hasMaxAttempts));
```

Comment:
Triggers the report of the existing standard error on the first error, then doesn't report the next 4 errors (while reloading the component) before sending a persistent-unknown error on the 5th attempt. Hopefully this means we can see if the player has recovered or not in bugsnag.

Reply:
Might be good to add the reload number to the metdata